pints/_mcmc/__init__.py

```diff
@@ -735,10 +784,17 @@ def run(self):
             # Write samples to disk
```

Comment:
- [ ] Check near line 686 ``# Store sample in memory``
- [ ] Add tests that do something like transform from a space [0, 1] to a space [10, 11], and make sure that all logged points are in [10, 11], whether that's logging to disk or to memory

Reply:
I have got that tests already!! See somewhere [here](https://github.com/pints-team/pints/pull/1165/files#diff-cf96e760e788f41c818773f7d42e7f15R763). The assert for all larger than zero ensure that is transformed back to the model space. Now I have added this explanation somewhere in the test...
```
# log-transform of the parameter in [0.01, 0.02] will always be         
# negative values, so checking it larger than zero make sure it's       
# transformed back to the model space
```

Reply:
Not sure what do you want me to check near line 686, that just assign the parameters from `tell()` to `samples`, and it is 'detransformed' before it's saved to disk and assign to `self._samples` (in memory).

Reply:
I will mark this as resolved. If I've missed your point, you can reopen this again.